cmd/daemon/prune.go

```diff
@@ -0,0 +1,101 @@
+package main
+
+import (
+	"fmt"
+	"os"
+	"os/signal"
+	"path/filepath"
+	"strings"
+	"syscall"
+
+	"github.com/pactus-project/pactus/cmd"
+	"github.com/pactus-project/pactus/store"
+	"github.com/pactus-project/pactus/util"
+	"github.com/spf13/cobra"
+)
+
+func buildPruneCmd(parentCmd *cobra.Command) {
+	pruneCmd := &cobra.Command{
+		Use:   "prune",
+		Short: "prune old blocks and transactions from client",
+		Long: "The prune command optimizes blockchain storage by removing outdated blocks and transactions, " +
+			"freeing up disk space and enhancing client performance. customize pruning criteria via flags to " +
+			"manage storage effectively.",
+	}
+	parentCmd.AddCommand(pruneCmd)
+
+	workingDirOpt := pruneCmd.Flags().StringP("working-dir", "w", cmd.PactusDefaultHomeDir(),
+		"a path to the working directory of node files")
+
+	pruneCmd.Run = func(_ *cobra.Command, _ []string) {
+		workingDir, _ := filepath.Abs(*workingDirOpt)
+		if util.IsDirNotExistsOrEmpty(workingDir) {
+			cmd.PrintErrorMsgf("The working directory is not exists: %s", workingDir)
+
+			return
+		}
+
+		conf, _, err := cmd.MakeConfig(workingDir)
+		cmd.FatalErrorCheck(err)
+
+		cmd.PrintLine()
+		cmd.PrintWarnMsgf("Warning: This command removes all your blocks and transactions and changes " +
+			"your node to a prune node. You cannot revert to a full node.")
```

Comment:
Bassicly they can revert to a full node. we can't add this to warning I think.
They can simply download the full data and start again, or use a backup.